logback-android/build.gradle

```diff
@@ -63,13 +63,14 @@ dependencies {
     testImplementation 'com.icegreen:greenmail:1.5.9'
     testImplementation 'dom4j:dom4j:1.6.1'
     testImplementation 'org.easytesting:fest-assert:1.4'
-    testImplementation "org.slf4j:integration:${slf4jVersion}"
+    testImplementation "org.slf4j:integration:2.0.0-alpha1" // don't seem to have any stable 2.x releases yet
```

Comment:
Was this change intentional? This PR changes `slf4jVersion` to `2.0.3` in `gradle.properties`, but then hard-codes it here.

Reply:
Not sure why I did that, will revert.

Reply:
ah actually its because they seem to not have a matching version available: https://mvnrepository.com/artifact/org.slf4j/integration - you can see the latest they offer is `2.0.0-alpha1`